itou/users/admin.py

```diff
@@ -664,6 +669,33 @@ def deactivate_view(self, request, user_pk):
         )
         return redirect(reverse("admin:users_user_change", kwargs={"object_id": user.pk}))
 
+    def reactivate_view(self, request, user_pk):
+        if not self.has_change_permission(request):
+            raise PermissionDenied
+        if request.method != "POST":
+            return HttpResponseNotAllowed(["POST"])
+        user = get_object_or_404(models.User, pk=user_pk)
+
+        if not user.can_be_reactivated():
+            return HttpResponseNotFound()
+
+        user.is_active = True
+        user.upcoming_deletion_notified_at = None
+        changed_fields = [
+            "is_active",
+            "upcoming_deletion_notified_at",
+        ]  # As a list to mimic Django change_message format
+        user.save(update_fields=changed_fields)
+        self.log_change(request, user, [{"changed": {"fields": changed_fields}}])
+
+        logger.info("user=%d reactivated", user.pk)
+        messages.success(request, format_html("Réactivation de l'utilisateur {user} effectuée.", user=user))
+        add_support_remark_to_obj(
+            user,
+            f"{timezone.now():%Y-%m-%d} ({request.user.get_full_name()}): Réactivation de l’utilisateur",
```

Comment:
Léo m'avait justement fait remarquer qu'on pouvait mettre l'heure également (mais clairement dans une autre PR que celle de désactivation 🙈 )
```suggestion
            f"{timezone.now():%Y-%m-%d %H:%M:%S} ({request.user.get_full_name()}): Réactivation de l’utilisateur",
```